<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ABBYY Automator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/application.png</normaloff>:/icons/application.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QFrame" name="fr_top_controls">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="inputs_layout">
         <item row="2" column="1">
          <widget class="QComboBox" name="cb_profile">
           <item>
            <property name="text">
             <string>Last used ABBYY settings</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Filetype</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="button_watch_browse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="LineEdit_DragDrop_Folder" name="le_watch_folder">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Watch folder</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Profile</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="rb_pdf">
              <property name="text">
               <string>Pdf</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_tiff">
              <property name="text">
               <string>Tiff</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_jpeg">
              <property name="text">
               <string>Jpeg</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="tb_refresh_profiles">
           <property name="toolTip">
            <string>Refresh profiles</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Output folder</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="LineEdit_DragDrop_Folder" name="le_output_folder">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="button_output_browse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pte_log">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QProgressBar" name="progress_bar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_save_errors">
        <property name="text">
         <string>Save Error Log</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_reset">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_start">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="StatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit_DragDrop_Folder</class>
   <extends>QLineEdit</extends>
   <header>custom_widgets</header>
  </customwidget>
  <customwidget>
   <class>StatusBar</class>
   <extends>QStatusBar</extends>
   <header>custom_widgets</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_watch_folder</tabstop>
  <tabstop>button_watch_browse</tabstop>
  <tabstop>le_output_folder</tabstop>
  <tabstop>button_output_browse</tabstop>
  <tabstop>cb_profile</tabstop>
  <tabstop>tb_refresh_profiles</tabstop>
  <tabstop>rb_pdf</tabstop>
  <tabstop>rb_tiff</tabstop>
  <tabstop>rb_jpeg</tabstop>
  <tabstop>pte_log</tabstop>
  <tabstop>button_save_errors</tabstop>
  <tabstop>button_reset</tabstop>
  <tabstop>button_start</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_start</sender>
   <signal>toggled(bool)</signal>
   <receiver>fr_top_controls</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
